week3/homework/src/index.js

```diff
@@ -1,3 +1,71 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const express = require('express');
+
+const readTodos = require('./readTodos');
+const createTodo = require('./createTodo');
+const deleteTodo = require('./deleteTodo');
+const updateTodo = require('./updateTodo');
+const readTodo = require('./readTodo');
+const clearTodos = require('./clearTodos');
+const markAsDone = require('./markAsDone');
+const markAsNotDone = require('./markAsNotDone');
+
+const app = express();
+
+app.use(express.json());
+
+try {
+  app.get('/todos', (request, response) => {
+    readTodos().then(todos => response.status(200).json(todos));
```

Comment:
I really like that you used `async` for the `readTodos` function. 
But why do you use then here. Wasn't the whole point of using `async` to use `await` instead of `then`?
How can you write this code so that is uses `await` instead of `then`?

Reply:
I got this error message when I used  await instead of then.

` const todos = await readTodos()`